components/ADempiere/FileRender/ExcelFile/index.vue

```diff
@@ -106,10 +106,15 @@ export default defineComponent({
     }
 
     function generateReaderData() {
+      console.log({
```

Comment:
@elsiosanchez please remove this console

Reply:
Ready. console removed